source/registry/rights/index.md

```diff
@@ -58,11 +56,10 @@ The process for having a new entry added to this registry is [described here][re
 
 This table summarizes the known rights statements and licenses available for use with the IIIF APIs.
 
-The URIs defined by [Creative Commons][org-cc-licenses] and [RightsStatements.org][org-rs-terms] are not listed here, but should be considered to be included.
```

Comment:
This one is important. Either we need to list all of the CC / RSs licenses or not list them and keep this assertion. The URIs in the PR aren't themselves licenses/statements.

Reply:
I like the idea of listing these two standards (esp. since we have no other to list at the moment), but could we have a column for each that says "for a full list of the CC license URIs, see...." rather than listing each license/statement URI? Then we don't have to worry about updating if the specific licenses/statements change over time -- and don't have to list them all here when they are listed elsewhere. Or we could list a few of the most common, then send folks to CC and RS for more.